Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,49 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses adds a new user to the system.
+
+By default, the user will be added to the `users` group (which has a `GID` of 100).
+
+This program must be run as root.
+
+## Options
+
+* `-u` _UID_: The user identifier for the new user. If not specified, an unused UID above `1000` will be auto-generated.
+* `-g` _GID_: The group identifier for the new user. If not specified, it will default to 100 (the **users** group).
+* `-s` _SHELLBIN_: The shell binary for this login. The default is `/bin/sh`. 
+* `-m`: Create a home directory for this new user at `/home/username`.
+* `-d` _HOMEPATH_: Set the home directory for this user to _HOMEPATH).
+* `-n` _GECOS_: GECOS information about this login. See [Wikipedia](https://en.wikipedia.org/wiki/Gecos_field) for more information.
+
+## Exit Values
+
+* 0 - Success
+* 1 - Couldn't update the password file
+* 3 - Invalid argument to option
+* 4 - UID already in use
+* 12 - Couldn't create home directory
+
+## Files
+
+* `/etc/passwd` - new user information (such as `UID` and `GID`) is appended to this file.
```

Comment:
Here and above: "UID" and "GID" are not code/identifiers, no need to make them monospace or otherwise emphasize